docs/quickstarts/ratify-on-alibabacloud.md

```diff
@@ -0,0 +1,357 @@
+# Ratify on Alibaba Cloud
+
+This guide will explain how to get up and running with Ratify on Alibaba Cloud using ACK and ACR. This will involve setting up necessary Alibaba Cloud resources, installing necessary components, and configuring them properly. Once everything is set up we will walk through a simple scenario of verifying the signature on a container image at deployment time.
```

Comment:
```suggestion
This guide will explain how to get up and running with Ratify on Alibaba Cloud using Alibaba Cloud Container Service for Kubernetes (ACK) and Aliabab cloud Container Registry (ACR). This will involve setting up necessary Alibaba Cloud resources, installing necessary components, and configuring them properly. Once everything is set up we will walk through a simple scenario of verifying the signature on a container image at deployment time.
```